graphlearn/include/config.h

```diff
@@ -85,6 +85,8 @@ DECLARE_STRING_GLOBAL_FLAG(ServerHosts)
 
 DECLARE_INT64_GLOBAL_FLAG(VineyardGraphID)
 DECLARE_STRING_GLOBAL_FLAG(VineyardIPCSocket)
+DECLARE_INT32_GLOBAL_FLAG(AttrStartIndex)
+DECLARE_INT32_GLOBAL_FLAG(AttrEndIndex)
```

Comment:
Different types of nodes has different attribute range (or, property names), and that cannot be a global flags.